ddtrace/logger.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+
+def configure_ddtrace_logger():
+    debug_log_level = os.environ.get("DD_TRACE_DEBUG", "false").lower() in ("true", "1")
+    log_path = os.environ.get("DD_TRACE_LOG_FILE", None)
+    max_file_bytes = os.environ.get("DD_TRACE_FILE_SIZE_BYTES", 15000000)
```

Comment:
Can you use `15 << 20` here with a comment `  # 15 MB` to match other usage internally

https://github.com/DataDog/dd-trace-py/blob/fb8dfa2f3/ddtrace/internal/writer.py#L53-L53

Reply:
```suggestion
    max_file_bytes = int(os.environ.get("DD_TRACE_FILE_SIZE_BYTES", 15000000))
```